src/main/java/homework2/Ticket.java

```diff
@@ -1,48 +1,48 @@
 package homework2;
 
-import java.util.Date;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
 import java.util.Random;
 
 public class Ticket {
-    private String ID;
```

Comment:
Please, leave string. It more suitable in this case) and just simple length validation

Reply:
ok